src/cascadia/WindowsTerminal/WindowUiaProvider.cpp

```diff
@@ -3,7 +3,7 @@
 #include "pch.h"
 
 #include "WindowUiaProvider.hpp"
-#include "../types/ScreenInfoUiaProvider.h"
+#include "../TerminalControl/ScreenInfoUiaProvider.hpp"

```

Comment:
So, this is needed in the ConHost accessibility model because the WindowUiaProvider. I'm not entirely sure if we need a WindowUiaProvider in Windows Terminal tbh, because XAML might be doing a lot of this for us already. That said, I'm hesitant to remove the WindowUiaProvider from the WindowsTerminal project until we get the following fixed:
- UIA Tree Navigation (#2102)
- Signals (#2447)

Removing the #include for now and updating TODOs to reflect this comment.